src/main/java/duke/exceptions/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke.exceptions;
+
+public abstract class DukeException extends Exception {
+    public DukeException(String msg) {
+        super(msg);
+    }
+}
```

Comment:
I like how you've created multiple exceptions that extend from `DukeException`, and also placed them in their own sub-package. Nice!